bin/fun-init.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/* eslint-disable quotes */
+
+'use strict';
+
+const program = require('commander');
+
+const examples =
+  `
+  Examples:
+
+    $ fun init
+    $ fun init foo/bar
+    $ fun init gh:foo/bar
+    $ fun init gl:foo/bar
+    $ fun init bb:foo/bar
+    $ fun init github:foo/bar
+    $ fun init gitlab:foo/bar
+    $ fun init bitbucket:foo/bar
+    $ fun init git+ssh://git@github.com/foo/bar.git
+    $ fun init hg+ssh://hg@bitbucket.org/bar/foo
+    $ fun init git@github.com/foo/bar.git
+    $ fun init hg@bitbucket.org/bar/foo
+    $ fun init https://github.com/foo/bar.git
+    $ fun init /path/foo/bar
+    $ fun init -n fun-app -V foo=bar /path/foo/bar
+  `;
+
+const parseVars = (val, vars) => {
+  /*
+   * Key-value pairs, separated by equal signs
+   * keys can only begin with letters, dollar signs, and underscores, other characters can only contain letters, numbers, and underscores
+   * values can be any character
+   */
+  const group = val.match(/(^[a-zA-Z$_][a-zA-Z\d_]*)=(.*)/);
+  vars = vars || {};
+  if (group) {
+    vars[group[1]] = group[2];
+  }
+  return vars;
+};
+
+program
+  .name('init')
+  .usage('[options] [location]')
+  .description('Initializes a new fun project')
+  .option('-o, --output-dir [outputDir]', 'where to output the initialized app into', '.')
```

Comment:
```suggestion
  .option('-o, --output-dir [path]', 'where to output the initialized app into', '.')
```
`commander.js` generates the option `program.outputDir` based on `--output-dir`. Changing to '[path]' may looks better.

Reply:
OK